### PR TITLE
Add the ability to use an external proxy for http(s) requests.

### PR DESCRIPTION
**Pull request by chetanyakan**
## Summary
- Add `request` package for making Http(s) requests.
- Add the ability to specify the proxy URL for requests.
- Create an Environment Variable for `PROXY_URL`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ that the resulting value includes only characters `[0-9a-f]`.
 
 Camo is configured through environment variables.
 
+* `PROXY_URL`: When provided, the requests to get the images will be proxied through this proxy_url.
```
Comment (jeheremy19p) [inline README.md:45]: *

**Review by jeheremy19p — CHANGES_REQUESTED** [review]
Close

**Comment by jeheremy19p** [comment]
PROXY_URL

**Comment by jeheremy19p** [comment]
Dependencias

**Comment by ErikMichelson** [comment]
The `request` package is deprecated. See [here](https://www.npmjs.com/package/request).
I suggest to close this PR.

**Pull request closed without merging**